src/views/Dashboard/Customer/RequestTestCheckout/index.vue

```diff
@@ -0,0 +1,162 @@
+                  
+
+            hr(class="ml-3 me-3 mb-5")
+            div(class="ml-5 mb-5 text-start")
+              b Payment Details:
+            div(class="ml-5 text-start")
+              v-row
+                v-col(cols="6") 
+                  div( style=" font-size: 12px;" ) Service Price:
+                v-col(cols="6") 
+                  div( style="font-size: 12px;" ) {{ testingPrice}} {{ currency}}
+              v-row
+                v-col(cols="6") 
+                  div( style="font-size: 12px;" ) Transaction Fee:
+                v-col(cols="6") 
+                  div( style="font-size: 12px;" ) {{ qcPrice }} {{ currency }}
+
+      template
+        div(align="center")
+          Button(
+            class="mt-16"
+            color="secondary" 
+            width="560"
+            height="38"
+            @click="onSubmit"
+            ) Proceed to payment
+
+</template>
+
+<script>
+
+import { mapState, mapMutations } from "vuex"
+import Stepper from '@/common/components/Stepper'
+import MenuCard from '@/common/components/MenuCard'
+import Button from '@/common/components/Button'
+
+
+export default {
+  name: "RequestTestCheckout",
+
+  components: {
+    Stepper,
+    MenuCard,
+    Button
+  },
+
+  data: () => ({
+    stepperItems: [
+      { number: 1, title: 'Select a location and Service Category', active: false },
+      { number: 2, title: 'Select a lab', active: false },
+      { number: 3, title: 'Checkout', active: true },
+      { number: 4, title: 'Payment', active: false },
+    ],
+    labDetail: "",
+    productDetail: "",
+    qcPrice: 0,
+    testingPrice: 0,
+    currency: ""
+  }),
+
+  async mounted () {
+    this.labDetail = this.labAccount._source.info
+    this.productDetail = this.product.info
+    this.qcPrice = this.product.info.prices_by_currency[0].additional_prices[0].value
+    this.testingPrice = this.product.info.prices_by_currency[0].price_components[0].value
+    this.currency = this.product.info.prices_by_currency[0].currency
+
+  },
+
+  computed: {
+    ...mapState({
+      country: (state) => state.lab.country,
+      city: (state) => state.lab.city,
+      category: (state) => state.lab.category,
+      labs: (state) => state.lab.labs,
+      labAccount: (state) => state.testRequest.lab,
+      product: (state) => state.testRequest.products
+    }),
+  },
+
+  methods : {
+    ...mapMutations({
+      setLabToRequest: "testRequest/SET_LAB",
+      setProductsToRequest: "testRequest/SET_PRODUCTS",
+    }),
+
+    async onSubmit () {
+      this.$router.push({ name: 'request-test-payment' })
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.request-checkout {
+    font-family: 'Raleway';
+    width: auto;
+    height: 732px;
+    background: #FFFFFF;
+}
+
+.title {
+  font-family: 'Raleway';
+  margin-top: 5%;
+  font-size: 20px;
+  line-height: 150%;
+  display: flex;
+  justify-content: center;
+}
+
+.card-test {
+  font-family: 'Raleway';
```

Comment:
do we need this font family? i think after my PR merge all font family will use Raleway as global default font family